server/storage/backend/verify.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backend
+
+import (
+	"os"
+	"runtime/debug"
+	"strings"
+
+	"go.uber.org/zap"
+)
+
+const (
+	ENV_VERIFY = "ETCD_VERIFY"
+	ENV_VERIFY_ALL_VALUE = "all"
```

Comment:
You can't or don't want to take dependency on server/verify.go ?

Reply:
can't, we have a dependency cycle :(

Reply:
SG. We should have TODO to move just: `shellVerify(foo)` helper out of the package.